src/js/intlTelInput.js

```diff
@@ -509,6 +510,7 @@ class Iti {
       // else let it bubble up to the top ("click-off-to-close" listener)
       // we cannot just stopPropagation as it may be needed to close another instance
       if (this.countryList.classList.contains('iti__hide') && !this.telInput.disabled && !this.telInput.readOnly) {
+        this.selectedFlag.setAttribute('aria-expanded', true);
```

Comment:
`aria-expanded` should only be applied to the element that toggles the expandable/collapsible content, not to the content itself. [Here are some examples from WCAG Working Group](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region) showing how to use `aria-expanded` correctly. It looks like this library already sets `aria-expanded` on the selected flag, which is the toggle, so we do not need this line:
```suggestion
```

Reply:
I just realized my mistake above. You’re _not_ setting `aria-expanded="true"` on the `countryList` element, you’re setting it on the `selectedFlag` element!

Reply:
Resolving this conversation, but leaving the other one pending.